### PR TITLE
PaperStreeMedia: Add FreeuseMilf

### PR DESCRIPTION
**Pull request by ryosaeba75**
Added freeusemilf.com to the list of website supported for this scrapper

### CODE REVIEW

**Comment by peolic** [comment]
You need to add the Studio name to the Studio section later in the file as well.

**Comment by Belleyy** [comment]
You need to change the date `# Last Updated February 25, 2020` for PaperStreetMedia.yml

File: scrapers/PaperStreetMedia.yml [hunk 1]
```diff
@@ -71,5 +73,5 @@ xPathScrapers:
                 teensloveanal: "Teens Love Anal"
                 teensloveblackcocks: "Teens Love Black Cocks"
                 thickumz: Thickumz
-                    
+
 # Last Updated February 25, 2020
```
Comment (Belleyy) [inline scrapers/PaperStreetMedia.yml:77]: Need to update the date

Comment (ryosaeba75) [reply scrapers/PaperStreetMedia.yml]: Done in https://github.com/stashapp/CommunityScrapers/pull/631/commits/af0b1b5557a19073bf2ca7ab3067add125eeb076

**Pull request merged**